src/components/ui4/RowUi4.tsx

```diff
@@ -86,13 +88,18 @@ export const RowUi4 = (props: Props) => {
           children
         )}
       </View>
-      <View style={styles.rightButtonContainer}>
-        {type === 'touchable' && <FontAwesome5 name="chevron-right" style={styles.tappableIcon} size={theme.rem(1)} />}
-        {type === 'editable' && <FontAwesomeIcon name="edit" style={styles.tappableIcon} size={theme.rem(1)} />}
-        {type === 'copy' && <FontAwesomeIcon name="copy" style={styles.tappableIcon} size={theme.rem(1)} />}
-        {type === 'delete' && <FontAwesomeIcon name="times" style={styles.tappableIcon} size={theme.rem(1)} />}
-        {type === 'questionable' && <SimpleLineIcons name="question" style={styles.tappableIcon} size={theme.rem(1)} />}
-      </View>
+      {isTouchable ? (
+        <>
+          <View style={styles.tappableIconMargin} />
```

Comment:
I would have attempted negative adding padding and margins to achieve the overlapping effect rather than a dummy component to take up space, but this works I guess.